spec/google/gax/operation_spec.rb

```diff
@@ -33,7 +33,6 @@
 require 'google/gax/settings'
 require 'google/gax/constants'
 require 'google/protobuf/any_pb'
```

Comment:
@landrito Please go ahead with this change. We cannot wait for the new grpc gem any more.